### DISCUSSION TITLE
Is there a rule for space inside blocks? E.g: {return 3} -> { return 3 }

### DISCUSSION THREAD

**Discussion by ftzi** [eslint/eslint#15615] [Ideas] [answered]
There is [space-before-blocks](https://eslint.org/docs/rules/space-before-blocks) but there doesn't seem to be a ~"space-inside-blocks", after the opening curly and before the closing curly (for blocks, not objects). Or there is?

**Comment by mdjermanovic** [comment, marked as answer]
There's [block-spacing](https://eslint.org/docs/rules/block-spacing) rule.

**Reply by ftzi** [reply]
MANY thanks! I love when there is a rule for what I want hehe.

I suggest adding it to the Related Rules in https://eslint.org/docs/rules/space-before-blocks

**Reply by mdjermanovic** [reply]
> I suggest adding it to the Related Rules in https://eslint.org/docs/rules/space-before-blocks

Good idea, thanks! I made a PR to update the docs https://github.com/eslint/eslint/pull/15625
